logging/logging-getting-started.adoc

```diff
@@ -0,0 +1,58 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="logging-getting-started"]
+= Getting started with {logging}
+include::_attributes/common-attributes.adoc[]
+:context: <logging-getting-started>
```

Comment:
```suggestion
:context: logging-getting-started
```
preview link is broken and I wonder if this is why - also, include the attributes above the title if you're using an attribute in the title

Reply:
I was scratching my head on that one, thanks!